Resources/Prototypes/Nyanotrasen/Actions/types.yml

```diff
@@ -165,7 +165,9 @@
   - type: InstantAction
     icon: { sprite: Nyanotrasen/Objects/Consumable/Food/candy.rsi, state: lollipop }
     useDelay: 120
-    event: !type:FabricateLollipopActionEvent
+    event:
+      !type:FabricateActionEvent
+      fabrication: FoodLollipop
```

Comment:
Move these actions to Prototypes/Actions/types.yml as well.

Reply:
within 3-5 business days